.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -288,6 +288,12 @@ jobs:
 
     - name: Copy v1 to pre-versioned bundles
       run: |
+        # Only execute if v1 files were created by the build job.
+        if ! [ -d "definitions_files/v1" ]; then
+           echo "error: no definition files for v1 were created: skipping..."
+           ls -lR definitions_files
+           exit 0
```

Comment:
I think leaving this step "green" sends the right message about how it was able to successfully upload one or more bundles, leaving "red" to "no bundle was found to upload".

Reply:
Could we add a conditional to the job yaml to explicitly mark the job as skipped? Maybe something like:
```
if: always() && needs."build-and-run (v1, 4.5.2)".result == 'success'
```
(that's probably not correct - I don't know how to get the job name from a matrix off the top of my head)

Reply:
Does GH actions have a built-in syntax to directly check individual matrix job in another job? I think I would have to use job outputs, which might be somewhat more complex than this. What do you think?

Reply:
Huh, apparently not. This is the best I could find: https://github.com/orgs/community/discussions/26822?sort=new#discussioncomment-3305794. So nevermind 🙂